surveys/templates/base.html

```diff
@@ -1,4 +1,4 @@
-{% load staticfiles %}
+{% load static %}
```

Comment:
Good catch! I noticed this but forgot to mention it. Where possible, fixing deprecation warnings is a good move.